e2e-tests/cypress/tests/integration/playbooks/channels/rhs/checklist_spec.js

```diff
@@ -164,7 +164,7 @@ describe('channels > rhs > checklist', () => {
             });
 
             // * Verify the expected error message.
-            cy.verifyEphemeralMessage('Failed to execute slash command /invalid');
+            cy.verifyEphemeralMessage('Failed to find slash command /invalid');
```

Comment:
I had already fixed this just before the mono-repo, but failed to migrate the changes across. Long story short, the Playbooks code now distinguishes errors between not finding a slash command from not executing it properly, and this test was supposed to catch the former.